core/deployment/src/main/java/io/quarkus/deployment/pkg/ManifestConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.deployment.pkg;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class ManifestConfig {
+
+    /**
+     * Manifest specification version.
+     */
+    @ConfigItem(defaultValue = "1.0")
+    public String version;
```

Comment:
Is it something that people change? I must admit that I'm not very accustomed to MANIFEST.MF fiddling so it's an honest question :).